documents/description.md

```diff
@@ -2,8 +2,21 @@
 
 ## Motivation
 
+Labanotation exists as commonly method used for notating movement. However, the language itself can be cumbersome and complicated to write and read. This makes it unaccessible to those who do not have serious training in transcribing and interpreting movement in Labanotation. As such, it would be beneficial to the movement analysis community to develop a language that retains the specifity of Labanotation while reducing the complexity. With this in mind, it would be useful to develop a language in which users can write out movement and see a simulation of that movement. Doing so would help both avoid the complexity of Labanotation and allow the user to see what their movement ideas look like in close to real time. 
+
```

Comment:
It would be good to link / show some examples of the notation, for the uninitiated.